.github/workflows/test.yml

```
@@ -0,0 +1,32 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  test:
+    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14.x, 16.x, 18.x, 19.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: ${{ matrix.node-version }}
+          check-latest: true
```

I'm not sure if the `check-latest` is needed, but could cache the NPM installs
```suggestion
          check-latest: true
          cache: npm
```

---

Oh, thanks @nschonni. Nice catch!

I've added the cache on the npm deps and removed the `check-latest` because it was actually a non-necessary check